### DISCUSSION TITLE
Question: Group Policy guidance for enabling Symbolic Links by default?

### DISCUSSION THREAD

**Discussion by JasonKleban** [git-for-windows/git#4846] [Q&A] [answered]
- [X] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

Is there a way to preset an installer option for domain-controlled workstations which will preset the Git for Windows installer's support for "Symbolic Links"?  Symbolic linking works well unless there's a machine that doesn't have that option checked on installation.  I would like to make it the default option value for all Git installations which might be executed.  Bonus if an existing installation will absorb this setting correctly upon rerunning the installer through on a repair or upgrade kind of scenario.

**Comment by JasonKleban** [comment]
I find 

```reg
[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Git_is1]
"Inno Setup CodeFile: Enable Symlinks"="Enabled"
```

Is this a reliable or is it system dependent?

**Reply by dscho** [reply, marked as answer]
As long as Git for Windows uses InnoSetup as its installer engine, it is reliable.

**Reply by JasonKleban** [reply]
Does the installer trust the state of this setting in the registry to decide to skip the system change if the install wizard thinks there's no options change by the user?  I think I had to uncheck > Finish and then rerun with check > Finish before it worked for an existing git installation.

**Reply by JasonKleban** [reply]
Is there any down side to graduating this to a standard feature of git for windows?

**Reply by dscho** [reply]
I am reluctant to promise that this will work until the end of time, but it is pretty robust and there is enough tooling around Git for Windows that relies on this that it is probably safe for you to use, too.
